### PR TITLE
updated doc wording, added CHANGELOG.md

### PR DESCRIPTION
**Pull request by cmarcelk**

### CODE REVIEW

**Comment by steveims** [comment]
the wording seems fine, but I don't know what all changed in this release.

do you manage a "release" branch?  could diff that to see what's changed.  (e.g. git flow is a common pattern: https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow)

File: relay/workflow.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 # without a specific handler, it will log to the console. Uncomment below to not send to console.
 # logger.addHandler(logging.NullHandler())
 
-version = "relay-sdk-python/2.0.1"
+version = "relay-sdk-python/2.0.0"
```
Comment (steveims) [inline relay/workflow.py:24]: should version be incremented?

Comment (cmarcelk) [reply relay/workflow.py:24]: Thought is to bump the version at the time of release. At that point we'll see what kind of content we have (hotfix vs addition vs breaking change), so we can figure out what the new version number should be.

Comment (steveims) [reply relay/workflow.py:24]: ah, so this is just reverting a previous 2.0.1?  got it.

**Review by steveims — APPROVED** [review]
lgtm

**Pull request merged**